app/marketing/models.py

```diff
@@ -407,3 +407,21 @@ def get_absolute_url(self):
 
     def __str__(self):
         return self.subject
+
+class UpcomingDate(SuperModel):
+    """Define the upcoming date model"""
+
+    title = models.CharField(max_length=255)
+    date = models.DateTimeField(db_index=True)
+    img_url = models.URLField(db_index=True, blank=True)
+    url = models.URLField(db_index=True)
+    comment = models.TextField(max_length=255, default='', blank=True)
+
+    @property
```

Comment:
todo in the future, make grants rounds, hackathons, workshops auto create these objects.